python/private/py_repositories.bzl

```diff
@@ -59,4 +59,10 @@ def py_repositories():
         sha256 = "2037875b9a4456dce4a79d112a8ae885bbc4aad968e6587dca6e64f3a0900cdf",
         strip_prefix = "rules_cc-0.0.9",
     )
+    http_archive(
```

Comment:
nit: I don't think we need this anymore if we are using the `rules_python_internal` repo.

Reply:
Done